src/main/steps/application-within-proceedings/utils.ts

```diff
@@ -358,3 +361,18 @@ export const getApplicationDetails = (
 
   return appDetails;
 };
+
+export const generateCancelLink = (
```

Comment:
May I know the rationale behind this method?

Reply:
I'm using this to create the cancel links with the correct application type and reason in the url